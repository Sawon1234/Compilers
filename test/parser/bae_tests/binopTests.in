fun testTimes(Integer a, Integer b) : Integer
{
	return a*b;
}

fun testSlash(Integer a, Integer b) : Integer
{
	return a/b;
}

fun testModulo(Integer a, Integer b): Integer
{
	return a%b;
}

fun testAdd(Integer a, Integer b) : Integer
{
	return a+b;
}

fun testSubtract(Integer a, Integer b) : Integer
{
	return a-b;
}