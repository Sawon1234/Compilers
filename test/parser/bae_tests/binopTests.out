fun testTimes < > (Integer a , Integer b ) : Integer { return a . times < > ( b ) } fun testSlash < > ( Integer a , Integer b ) : Integer { return a . slash ( b ) ; } fun testModulo < > ( Integer a , Integer b ) : Integer { a . modulo ( b ) ; } fun testAdd < > ( Integer a , Integer b ) : Integer { return a . plus ( b ) ; } fun testSubtract ( Integer a , Integer b ) : Integer { return a . subtract ( b ) ; }